tools/make/kube.mk

```diff
@@ -43,6 +43,8 @@ CONTROLLERGEN_OBJECT_FLAGS :=  object:headerFile="$(ROOT_DIR)/tools/boilerplate/
 manifests: $(tools/controller-gen) generate-gwapi-manifests ## Generate WebhookConfiguration and CustomResourceDefinition objects.
 	@$(LOG_TARGET)
 	$(tools/controller-gen) crd:allowDangerousTypes=true paths="./api/..." output:crd:artifacts:config=charts/gateway-helm/crds/generated
+	# save a copy of all crds for local resource validator.
+	find charts/gateway-helm/crds -name "*.yaml" -exec cat {} + > internal/gatewayapi/resource/crd/gateway-crds.yaml
```

Comment:
can we do w/o this extra copy ?

Reply:
Cannot think of better way to avoid copy. 

`embed` enforce the path must not contain `.` or `..` path elements nor begin with a leading slash, so it means the `gateway-crds.yaml` must under the `resource` pkg.

Reply:
I would instead prefer if we created a top level `embed.go` in the root dir, so we eliminated this copy

Reply:
brilliant idea!

Reply:
im not very proud of it 🙈 , it make the root dir a little impure, but its reduces the amount of diffs generated every time a CRD changes